cpp/src/nvgraph/include/nvgraph_cusparse.hxx

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2020, NVIDIA CORPORATION.
```

Comment:
We have 3 cuSparse wrappers... This needs to be reconciled with the wrapper in `cpp/src/utils` and the one in RAFT. It doesn't have to be in this PR though, I'll create a separate issue.